pkg/azure/blob/credentials.go

```diff
@@ -47,9 +48,12 @@ func GetCredentials(cfg azureconfig.Config, l hclog.Logger) (CredentialSet, erro
 
 	connString := os.Getenv(credsEnv)
 	if connString == "" {
-		cred, err := GetCredentialsFromCli(cfg, l)
+		cred, useCli, err := GetCredentialsFromCli(cfg, l)
+		if !useCli {
+			return CredentialSet{}, errors.Errorf("environment variable %s containing the azure storage connection string was not set:\n%#v", credsEnv, cfg)
```

Comment:
You marked this (below) as resolved but are still not wrapping errors using `github.com/pkg/errors`.

https://github.com/getporter/azure-plugins/pull/30#discussion_r518877175

Reply:
Sorry, no idea why I didn't do this 😦